### DISCUSSION TITLE
【招募】防卫军招募您！

### DISCUSSION THREAD

**Discussion by Soldier11-ObsidianBarracks** [share121/inter-knot#3338] [其他]
我们需要二十位武器方面研究人员
我们照收新艾丽都大学武器研制专业，材料学专业，以太环境监测＆测试专业等理工方面
以太适应性需要达到60及以上
学历本科以上（不包括本科）
其余请到斯科特哨站找到罗恩了解详情

**Comment by GrunderIndustries** [comment]
创立者工业可提供您所需的一切人员与技术

**Reply by Soldier11-ObsidianBarracks** [reply]
尊嘟假嘟O.o
那就来斯科特哨站找我

**Reply by GrunderIndustries** [reply]
创立者工业已委托一名代理人前往哨站交涉
